libkbfs/folder_branch_ops.go

```diff
@@ -6073,6 +6073,73 @@ func (fbo *folderBranchOps) onMDFlush(bid BranchID, rev kbfsmd.Revision) {
 	}()
 }
 
+// TeamNameChanged implements the KBFSOps interface for folderBranchOps
+func (fbo *folderBranchOps) TeamNameChanged(
+	ctx context.Context, tid keybase1.TeamID) {
+	fbo.log.CDebugf(ctx, "Team name changed for team %s", tid)
+
+	go func() {
```

Comment:
I'll break it out into a separate function.

It's a goroutine because we shouldn't block the service notification RPC while waiting on the lock accesses and outgoing service calls (which could lead to deadlock potential).